doc/integrator/c2cwsgiutils.rst

```diff
@@ -1,15 +1,49 @@
 .. _integrator_c2cwsgiutils:
 
-c2cwsgiutils
-============
+C2C WSGI Utils
+==============
 
-c2cwsgiutils is a framework assisting in development, integration and administration of WSGI applications.
-c2cgeoportal uses c2cwsgiutils.
+ ``c2cwsgiutils`` is a framework assisting in development, integration and administration of WSGI applications.
+``c2cgeoportal`` uses ``c2cwsgiutils``.
 See its `documentation <https://github.com/camptocamp/c2cwsgiutils/#camptocamp-wsgi-utilities>`__.
 
 The entry point URL is ``<application main URL>/c2c``, where you will have a dashboard with the
-c2cwsgiutils and c2cgeoportal services.
+``c2cwsgiutils`` and ``c2cgeoportal`` services.
 
-To use the protected tools, you should at least set ``C2C_SECRET`` to a secret password in the ``env.project`` file.
+The ``invalidate`` view is also protected with this sec
```

Comment:
```suggestion
The ``invalidate`` view is also protected with this secret.
```
But we do not know anymore what is "this secret".
Seems that you should remove completely this sentence here.

Reply:
=>
The ``invalidate`` view is protected with the same authentication, see bellow.